pyproject.toml

```diff
@@ -54,6 +54,7 @@ pylint = "^3.2.2"
 pytest = "^8.2.1"
 pytest-cov = "^5.0.0"
 types-python-dateutil = "^2.9.0.20240316"
+pytest-env = "^1.1.3"
```

Comment:
Use this package for env vars during testing to avoid any ENV VAR mocking requirements.